SharpPcap/LibPcap/LibPcapSafeNativeMethods.Interop.cs

```diff
@@ -33,58 +34,45 @@ namespace SharpPcap.LibPcap
     /// Per http://msdn.microsoft.com/en-us/ms182161.aspx 
     /// </summary>
     [SuppressUnmanagedCodeSecurity]
-    internal static class Windows
+    internal static partial class LibPcapSafeNativeMethods
     {
+        // NOTE: For mono users on non-windows platforms a .config file is used to map
```

Comment:
I'm not sure how close this is to doing away. NativeLibraries is from the mono dll mapping use case so we might not need it. Actually it looks like we are all set to remove mono specific stuff in the near future, see https://www.mono-project.com/news/2020/08/24/native-loader-net5/

Reply:
Opened a PR that matches with our switch to NativeLibrary usage.